bot.py

```diff
@@ -395,6 +404,34 @@ def get_repo_info(self, channel_id):
         else:
             raise Exception("Configuration error: unknown project type {}".format(repo_info.project_type))
 
+    async def hotfix_command(self, command_args):
+        """
+        Start a hotfix with the commit hash provided
+
+        Args:
+            command_args (CommandArgs): The arguments for this command
+        """
+        repo_info = command_args.repo_info
+        repo_url = repo_info.repo_url
+        org, repo = get_org_and_repo(repo_url)
+
+        release_pr = await get_release_pr(github_access_token=self.github_access_token, org=org, repo=repo)
+        if release_pr:
+            await self.say(
+                channel_id=repo_info.channel_id,
+                text=f"There is a release already in progress: {release_pr.url}. Close that first!"
+            )
+            raise ReleaseException(f"There is a release already in progress: {release_pr.url}. Close that first!")
+
+        async with init_working_dir(self.github_access_token, repo_info.repo_url):
+            last_version = update_version("9.9.9")
+
+        _, new_patch = next_versions(last_version)
+
+        command_args.args.insert(0, new_patch)
+
+        await self._web_application_release(command_args)
```

Comment:
Instead of editing `command_args`, can we change the arguments accepted by `_web_application_release`?